integration/scripts/integration.sh

```diff
@@ -25,17 +25,17 @@ usage() {
 }
 
 build_containers() {
-    bash -c "${DC} -f docker-compose.yml -f docker-compose.xpu.yml -f docker-compose.otel.yml -f docker-compose.spdk.yml -f docker-compose.pxe.yml build --parallel"
+    bash -c "${DC} -f docker-compose.yml -f docker-compose.xpu.yml -f docker-compose.otel.yml -f docker-compose.spdk.yml -f docker-compose.api.yml -f docker-compose.pxe.yml build --parallel"
```

Comment:
It might be time to turn this into a function, since we have to copy/paste this long prefix all over the place...

Something like run_dc_command("build --parallel")

Reply:
@seroyer see #106